src/game/structures.js

```diff
@@ -1088,6 +1088,118 @@ exports.make = function(_runtimeData, _intents, _register, _globals) {
         return name;
     });
 
+    StructureSpawn.prototype.spawnCreep = register.wrapFn(function spawnCreep(body, name, options = {}) {
```

Comment:
Might be a good idea to check in new options object with `_.isObject(options)`. If false -> return C.ERR_INVALID_ARGS

Reply:
This also should be tested if it's transpiled correctly to ES5 code, since it goes to the simulation browser code which doesn't support ES6 features.

Reply:
@artch interesting. I didn't realize. What are you using to transpile?

Reply:
Added the safeguard on options. Once I know what you guys are using for transpilation, I can check its compatibility.

Alternatively, I can just swap it with a simple default behavior.